Ginger/GingerCoreNET/Drivers/CoreDrivers/Mobile/Appium/GenericAppiumDriver.cs

```diff
@@ -960,25 +954,79 @@ private void GenElementHandler(ActGenElement act)
             }
         }
 
+        private string GetAppPackage(ActMobileDevice act)
+        {
+            string appPackage = null;
+            if (string.IsNullOrEmpty(act.ActionAppPackage.ValueForDriver) || act.ActionAppPackage.ValueForDriver.ToLower().Trim() == "default")
+            {
+                if (DevicePlatformType == eDevicePlatformType.Android)
+                {
+                    appPackage = AppiumCapabilities.Where(x => x.Parameter == "appPackage" || x.Parameter == "appium:appPackage").FirstOrDefault().Value;
```

Comment:
Please use Find if its list or FirstOrDefault instead of where e.g. AppiumCapabilities.FirstOrDefault(x => x.Parameter == "appPackage" || x.Parameter == "appium:appPackage").Value;